### ISSUE TITLE
Table based pods fail to update database when changing type

### ISSUE THREAD

**Issue by jamesgol** [pods-framework/pods#5506]
**Describe the bug**
Reported by 'Rob Earls' in Slack.  Starting with a Table based Pod with a relationship to User, switching the field to 'Plain Text' does not update the database properly.

Pull request incoming

**To Reproduce**

**Expected behavior**

**Screenshots**

## Pods Version
Please provide the version of Pods you are running: 

## WordPress Environment

<details>

Copy and paste your Tools > Site Health > Info. Use the 'Copy Site Info to Clipboard' and paste the results over this line.

</details>

## Pods Package Export (helpful!)

```
Copy and Paste the JSON Export from **Pods Admin, Migrate: Packages, Export** in WordPress admin here
```

**Additional context**

**Possible Workaround**
If you have discovered a workaround, please include it below.

**Comment by jamesgol** [comment]
Another oddity I uncovered when doing this (related to the Mysql settings WordPress uses or possibly a bug in the Mysqli library), is if you go the opposite direction the 'ALTER TABLE...DROP COLUMN' will fail and the query returns false but no error message is captured by wp-db.php so Pods silently ignores the error.

**Comment by JoryHogeveen** [comment]
@jamesgol Can this be closed since #5507 is merged?

**Comment by sc0ttkclark** [comment]
Fixed via https://github.com/pods-framework/pods/pull/5507
